### PR TITLE
Implement sub-menu of account page and routes.

### PR DESCRIPTION
**Pull request by abouolia**

### CODE REVIEW

File: packages/renderer/src/components/molecules/AccountSidebar/AccountSidebar.test.tsx [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+import { render } from "@testing-library/react";
+import { Routes, HashRouter, Route } from "react-router-dom";
+import { AppProviders } from "@atoms/AppProviders";
+import { AccountSidebar } from "./AccountSidebar";
+
+describe("molecules::AccountSidebar", () => {
+  it("renders without exploding", () => {
+    const { asFragment } = render(
+      <AppProviders>
+        <HashRouter>
```
Comment (schowdhuri) [inline packages/renderer/src/components/molecules/AccountSidebar/AccountSidebar.test.tsx]: Let's refactor `AppProviders` to include `HashRouter`

File: packages/renderer/src/components/molecules/AccountSidebar/_constants.ts [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+import { useIntl } from "react-intl";
+
+export const WIDTH = 210;
+
+export interface AccountSidebarItem {
+  label: string;
+  route: string;
+}
+
+export const useGetAccountSidebarMenu = () => {
```
Comment (schowdhuri) [inline packages/renderer/src/components/molecules/AccountSidebar/_constants.ts]: The hook can be extracted out to a file in the same folder

**Review by schowdhuri — CHANGES_REQUESTED** [review]
- Please remove package-lock.json. The project uses yarn.
- Please run `yarn license`

**Comment by abouolia** [comment]
I know I use yarn too just pushed it by mistake after `yarn license && git add -A`, and quickly fixed the mistake.

> * Please remove package-lock.json. The project uses yarn.
> * Please run `yarn license`

File: packages/renderer/src/components/molecules/AccountSidebar/_constants.ts (hunk 2 above)
Comment (abouolia) [reply packages/renderer/src/components/molecules/AccountSidebar/_constants.ts]: Maybe in `hooks.ts` file is much better.

File: packages/renderer/src/components/molecules/AccountSidebar/AccountSidebar.test.tsx (hunk 1 above)
Comment (abouolia) [reply packages/renderer/src/components/molecules/AccountSidebar/AccountSidebar.test.tsx]: @schowdhuri refactored

File: packages/renderer/src/components/molecules/AccountSidebar/_constants.ts (hunk 2 above)
Comment (abouolia) [reply packages/renderer/src/components/molecules/AccountSidebar/_constants.ts]: @schowdhuri Moved to `_hooks.ts` on same parent folder.

**Review by schowdhuri — APPROVED** [review]
Love the super clean code! 🔥

**Pull request merged**